pkgs/applications/misc/rivercarro/default.nix

```diff
@@ -46,4 +51,3 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ kraem ];
```

Comment:
not sure what is wrong with aarch64-linux but I would suggest to mark it broken on aarch64-linux

```
LLVM Emit Object... Semantic Analysis [2361] gfx1036... Semantic Analysis [2719] btver2... Semantic Analysis [2977] getOrPut... Semantic Analysis [3263] reallocAtLeast... Semantic Analysis [3347] eventfd... Semantic Analysis [3476] sizes... Semantic Analysis [3609] isEmpty... Semantic Analysis [3722] exeFileExtSimple... Semantic Analysis [3828] count... Semantic Analysis [3948] debug... Semantic Analysis [4007] iterator... Semantic Analysis [4110] grow... Semantic Analysis [4179] minimal_capacity... Semantic Analysis [4232] allocatedSlice... Semantic Analysis [4256] minimal_capacity... LLVM Emit Object... LLVM Emit Object... Semantic Analysis [4266] LLVM Emit Object... LLVM Emit Object... LLD Link... LLD Link... LLVM Emit Object... LLVM Emit Object... Semantic Analysis [1613] /build/source/zig-cache/zig-wayland/common.zig:103:21: error: cast increases pointer alignment
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/zig-cache/zig-wayland/common.zig:103:68: note: '?*const anyopaque' has alignment '1'
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                                                                   ^~~~~~~~~~~~~~
/build/source/zig-cache/zig-wayland/common.zig:103:30: note: '*const fn(*wayland_client.Registry, wayland_client.Registry.Event, *main.Context) void' has alignment '4'
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    setListener: /build/source/zig-cache/zig-wayland/wayland_client.zig:159:56
    main: /build/source/src/main.zig:450:13
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
/build/source/zig-cache/zig-wayland/common.zig:103:21: error: cast increases pointer alignment
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/zig-cache/zig-wayland/common.zig:103:68: note: '?*const anyopaque' has alignment '1'
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                                                                   ^~~~~~~~~~~~~~
/build/source/zig-cache/zig-wayland/common.zig:103:30: note: '*const fn(*river_layout_v3_client.LayoutV3, river_layout_v3_client.LayoutV3.Event, *main.Output) void' has alignment '4'
                    @ptrCast(*const fn (*Obj, Payload, Data) void, implementation)(
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
error: rivercarro...
error: The following command exited with error code 1:
```

Reply:
Yeah, I'm not sure why it's failing now on aarch64-linux either. I've marked it as broken and will let someone who actually uses this package decide if they care to fix it.